skimage/measure/_regionprops.py

```diff
@@ -166,6 +167,31 @@ def extent(self):
         return self.moments[0, 0] / (rows * cols)
 
     @property
+    def feret_diameter(self):
+        from ..morphology.convex_hull import convex_hull_image
+        from ._find_contours import find_contours
+        
+        label_image = self._label_image
+        label = self.label
+        
+        max_feret = 0.0
+        identity_convex_hull = convex_hull_image(label_image == label)
+        coordinates = np.vstack(find_contours(identity_convex_hull, 0.5, fully_connected = 'high'))
+        
+        n = len(coordinates)
```

Comment:
This could be replaced with `pdist`. At the least, do `for b in range(0, a):` to avoid duplicate computation.